src/Admin/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -234,8 +242,24 @@ await _referenceEventService.RaiseEventAsync(new ReferenceEvent(ReferenceEventTy
     public async Task<IActionResult> Delete(Guid id)
     {
         var organization = await _organizationRepository.GetByIdAsync(id);
+
         if (organization != null)
```

Comment:
Please invert this `if` and return early to avoid unnecessary nesting.

Reply:
Inverted here: https://github.com/bitwarden/server/pull/3996/commits/c0f6e78cfd8b2ddaded880af79590c81054ce361